client/dashboard/components/callout-overlay/style.scss

```diff
@@ -6,11 +6,17 @@
 	width: calc( 100% - 2 * $grid-unit-20 );
 	top: 0;
 	inset-inline-start: 0;
-	padding: 161px $grid-unit-20;
+	padding-top:max(7vh, 40px);
+	padding-top: max(7dvh, 40px);
```

Comment:
Duplicate rule?

Reply:
No `dvh` is not supported across the board. `vh` is the fallback. Spacing issue though! 😴